Spigot-API-Patches/0161-Add-more-Witch-API.patch

```diff
@@ -37,6 +37,16 @@ index aa88aede6c4e66a608a63d07bc66d60357b0bee9..cd8d0a20e52ad35f659175e0d24a1dc0
 +    int getPotionUseTimeLeft();
 +
 +    /**
++     * Set time remaining (in ticks) that the Witch is drinking a potion.
++     * <p>
```

Comment:
Where's the closing tag?

Reply:
Missing close tag is legal (& standard-ish) in Javadocs. Don't ask why, though :p